spec/features/holds_spec.rb

```diff
@@ -120,4 +120,20 @@
       expect(page).to have_css '#hold3911148'
     end
   end
+
+  context 'when site is in maintenance mode' do
+    after do
+      Settings.maintenance_mode = false
+    end
+
+    it 'makes a child checkbox with that name', js: true do
```

Comment:
description needs to be updated here as well?

Reply:
yup

Reply:
only this description left to update ☝️